diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -183,6 +184,22 @@ fun ASTNode.isAnonymousFunction(): Boolean {
     return this.getIdentifierName() == null
 }
 
+/**
+ * Checks if the function has boolean return type
```

Comment:
please add kdoc `@return`

Reply:
why diktat didnt require it by itself?

Reply:
Done. There was a `KDOC_WITHOUT_RETURN_TAG` suppress.

Reply:
discussed offline: `AstNodeUtils.kt` has suppression for this rule on whole file.